packages/pn-commons/src/components/NotificationDetail/NotificationPaymentRecipient.tsx

```diff
@@ -34,6 +36,14 @@ type Props = {
     unwrap: () => Promise<PaymentAttachment>;
   };
   onPayClick: (noticeCode?: string, creditorTaxId?: string, amount?: number) => void;
+  handleReloadPayment: (payment: Array<PaymentDetails | NotificationDetailPayment>) => void;
```

Comment:
@alessandrogelmi ho risolto dei conflitti sia su questa pagina che su notification Detail, legati alla gestione dei pagamenti. In particolare handleReloadPayment non sembra referenziato.
Puoi controllare che nel merge non si siano persi gli ultimi fix/sviluppi?